app/workers/analytic_ingester_worker.rb

```diff
@@ -3,40 +3,80 @@
 class AnalyticIngesterWorker
   @queue = :analytic_ingester
 
-  def self.perform(fastfile_id, error, crash)
+  def self.perform(fastfile_id, error, crash, launches, timestamp_seconds)
     start = Time.now
 
-    completion_status =  crash.present? ? 'crash' : ( error.present? ? 'error' : 'success')
-    analytic_event_body = {
-      analytics: [{
-        event_source: {
-          oauth_app_name: 'fastlane-enhancer',
-          product: 'fastlane_web_onboarding'
-        },
-        actor: {
-          name:'customer',
-          detail: fastfile_id
-        },
-        action: {
-          name: 'fastfile_executed'
-        },
-        primary_target: {
-          name: 'fastlane_completion_status',
-          detail: completion_status
-        },
-        millis_since_epoch: Time.now.to_i * 1000,
-        version: 1
-      }]
-    }.to_json
+    completion_status = crash.present? ? 'crash' : ( error.present? ? 'error' : 'success' )
+    analytics = []
+    if fastfile_id.present? && launches.size == 1 && launches['fastlane']
+      analytics << fastlane_web_onboarding_event(fastfile_id, completion_status, timestamp_seconds)
+    end
+
+    launches.each do |action, count|
+      action_completion_status = action == crash ? 'crash' : ( action == error ? 'error' : 'success' )
+      analytics << fastlane_execution_event(action, count, action_completion_status, timestamp_seconds)
+    end
+
+    analytic_event_body = { analytics: analytics }.to_json
 
     puts "Sending analytic event: #{analytic_event_body}"
 
-    Faraday.new(:url => ENV["ANALYTIC_INGESTER_URL"]).post do |req|
+    response = Faraday.new(:url => ENV["ANALYTIC_INGESTER_URL"]).post do |req|
       req.headers['Content-Type'] = 'application/json'
       req.body = analytic_event_body
     end
 
     stop = Time.now
+
+    puts "Analytic ingester response was: #{response.status}"
     puts "Sending analytic ingester event took #{(stop - start) * 1000}ms"
   end
+
+  def self.fastlane_web_onboarding_event(fastfile_id, completion_status, timestamp_seconds)
```

Comment:
This method uses a different naming convention as the ones we wrote in _refresher_, definitely not a big deal, but we had `event_for_<thing>` rather than `<thing>_event`.